nikola/__main__.py

```diff
@@ -238,16 +238,13 @@ def __init__(self, nikola, quiet=False):
         self.quiet = quiet
 
     def load_tasks(self, cmd, opt_values, pos_args):
+        DOIT_CONFIG = config.get('DOIT_CONFIG', {})
         if self.quiet:
-            DOIT_CONFIG = {
-                'verbosity': 0,
-                'reporter': 'zero',
-            }
+            DOIT_CONFIG['verbosity'] = 0
+            DOIT_CONFIG['reporter'] = 'zero'
         else:
-            DOIT_CONFIG = {
-                'reporter': ExecutedOnlyReporter,
-                'outfile': sys.stderr,
-            }
+            DOIT_CONFIG['reporter'] = ExecutedOnlyReporter
```

Comment:
So then we can't just pop it out and throw it away in ``__init__``  It has to stay somewhere so that ``load_tasks`` can get to it.